4.3/README.md

```diff
@@ -117,6 +127,89 @@ docker run -it --shm-size=1g \
   --name nominatim \
   mediagis/nominatim:4.3
 ```
+### Configuration Example
```

Comment:
I think this is useful but I would like to see it moved to a separate file and then linked to from the main readme.

Reply:
I managed to make the file, but i dont know what i can do to link the file inthe readme

Reply:
The Markdown link syntax is simply `[examples](example.md)`

Reply:
Yeah gpt saved me again - but just at the end for links in forks etc.

Reply:
Should i additionally put a short sentence with the link in the readme?

Reply:
Yes, that's fine.

Reply:
Then also update the TOC at the top.

Reply:
Done!